nano/node/json_handler.cpp

```diff
@@ -3910,6 +3910,12 @@ void nano::json_handler::telemetry ()
 			uint16_t port;
 			if (!nano::parse_port (*port_text, port))
 			{
+				if (!address_text->empty () && address_text->front () == '[' && address_text->back () == ']')
```

Comment:
Strange that it's the first time running into this, should we extract a function?
@SergiySW bootstrap RPC also uses make_address_v6, should we do the same there?

Reply:
We had `parse_port` and `parse_port_and_address` in common so I've moved it there.
I think this is because in most places we don't use `[]`